### ISSUE TITLE
VSCodium support

### ISSUE THREAD

**Issue by KaspianDev** [gitpod-io/gitpod#10667]
## Is your feature request related to a problem? Please describe
I am not a big fan of binaries that microsoft redistributes, they contain some crap I don't really want like telemetry which makes it nonfree software. VSCodium is a good approach to fix it. It builds open source binary without microsoft stuff and uses open source extension store.
However, gitpod "open in vscode" does not open workspace in VSCodium, simply nothing happens.
So here is my suggestion, open workspace in VSCodium

## Describe alternatives you've considered
I tried ssh but it wasn't as smooth and easy.

**Comment by KaspianDev** [comment]
This extension seems to do the thing (ssh extension replica, but OSS).
 - https://github.com/xaberus/vscode-remote-oss#readme=
 And it also seems most of the features from original work without change.

**Comment by wvffle** [comment]
:eyes:

**Comment by axonasif** [comment]
Not stale.

**Comment by astrojuanlu** [comment]
_activity_

**Comment by GitMensch** [comment]
Wouldn't it be enough to add "VSCodium (Desktop)" to the options that can be chosen, then do the exact same thing as with "VSCode (Desktop)" but instead of using a `vscode://` url use `vscodium://`, as per https://github.com/VSCodium/vscodium/issues/1126#issuecomment-1156711667?

**Comment by axonasif** [comment]
@GitMensch I think the issue is not about establishing a SSH connection, that's already doable (manually), it's about detecting user interaction as heartbeat so that the workspace doesn't timeout.

**Comment by GitMensch** [comment]
t would be nice to not need to start that manually - is there a way I can get GitPod Workspaces to start with a vscodium URL instead (or at least get the vscode URL to clipboard for testing so I can manually change that to the other scheme)?

What part does the heartbeat on the "VSCode Desktop" option? The GitPod extension / an internal SSH connection / something else?

**Comment by KaspianDev** [comment]
All that's needed to support it should be respecting vscodium's protocol (because it has a different name) but I might be wrong. Just commenting.

**Comment by jeanp413** [comment]
You should be able to connect using vscodium now, steps:
- Install open-remote-ssh extension https://open-vsx.org/extension/jeanp413/open-remote-ssh
- Install gitpod extension https://open-vsx.org/extension/gitpod/gitpod-desktop
- Login using gitpod, [see docs](https://www.gitpod.io/docs/references/ides-and-editors/vscode#prerequisites)
- Connect to an existing workspace using the workspace explorer view 
![image](https://github.com/gitpod-io/gitpod/assets/25115070/c242ead1-17e6-4860-9e2c-2a20ef8242c8)
- Creating new workspaces using the gitpod extension is not yet supported so you'll need to go to the gitpod dashboard to create a new workspace first and then connect from the workspace explorer view in vscodium

**Comment by ENate** [comment]
@jeanp413 Thanks for the info. I could not connect by following the instructions you posted. Will this work if I have vscode installed or it is best to remove vscode and follow the steps for codium? I saw that an access token was provided but it is not possible to add the token because the 'token option' for Gitpod is only available for private beta users.

**Comment by jeanp413** [comment]
> Will this work if I have vscode installed or it is best to remove vscode and follow the steps for codium?

yes, it should be fine to have both vscode and vscodium installed

>  I could not connect by following the instructions you posted.

@ENate  Could you give more info about what step you failed to complete successfully? maybe share gitpod and/or remote-ssh extension logs

**Comment by GitMensch** [comment]
@jeanp413 connecting from vscodium works fine now, can you have a look at letting vscodium be opened by GitPod online, too?

> It would be nice to not need to start that manually - is there a way I can get GitPod Workspaces to start with a vscodium URL instead (or at least get the vscode URL to clipboard for testing so I can manually change that to the other scheme)?
